tests/integration/test_e2e.py

```diff
@@ -0,0 +1,35 @@
+'''End-to-end/smoke tests for htetl'''
+import luigi
+import luigi.mock
+
+import htetl.tasks.main as tasks_main
+from .. import test_data
+from .. import integration
+
+
+class TestITLoadEntityIds(object):
+
+    '''Smoketest whole ETL pipeline'''
+
+    def setup(self):
+        self.config = luigi.configuration.LuigiConfigParser.instance()
+        self.task = tasks_main.LoadEntityIds()
+
+    def tearDown(self):
+        engine = integration.sqlalchemy_engine()
+        engine.execute('DROP TABLE IF EXISTS "BackpageEntities"')
+        engine.execute('DROP TABLE IF EXISTS backpageentities')
```

Comment:
Is there a bug where we're creating both of these?

Reply:
No, it's a safe guard in case we do and don't clean up.